docs/user/private-app-repository.md

```diff
@@ -0,0 +1,105 @@
+
+<img src="../img/chartmuseum-deploy-form.png" alt="ChartMuseum Deploy Form" width="600px">
+
+### ChartMuseum: Upload a Chart
+
+Once the chart is deployed you will be able to upload a chart. In one terminal open a port-forward tunnel to the application:
+
+```console
+$ export POD_NAME=$(kubectl get pods --namespace default -l "app=chartmuseum" -l "release=my-chartrepo" -o jsonpath="{.items[0].metadata.name}")
+$ kubectl port-forward $POD_NAME 8080:8080 --namespace default
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+```
+
+And in a different terminal you can push your chart:
+
+```console
+$ helm package /kubeapps/chart/kubeapps
+Successfully packaged chart and saved it to: /kubeapps/chart/kubeapps/kubeapps-1.0.0.tgz
+$ curl --data-binary "@kubeapps-1.0.0.tgz" http://localhost:8080/api/charts
+{"saved":true}
+```
+
+### ChartMuseum: Configure the repository in Kubeapps
+
+For adding your private repository go to `Configuration > App Repositories` in Kubeapps and click on "Add App Repository". You will need to add your repository using as URL the service of the application. This will be `<release_name>-chartmuseum.<namespace>:8080`:
+
+<img src="../img/chartmuseum-repository.png" alt="ChartMuseum App Repository" width="600px">
+
+Once you create the repository you can click in the link for the specific repository and you will be able to deploy your own application using Kubeapps.
+
+### ChartMuseum: Authentication/Authorization
+
+It is possible to configure ChartMuseum to use authentication with two different mechanism:
+
+- Using HTTP [basic authentication](https://chartmuseum.com/docs/#basic-auth) (user/password). To use this feature, it's needed to:
+  - Specify the parameters `secret.AUTH_USER` and `secret.AUTH_PASS` when deploying the ChartMuseum.
+  - Change the URL of the App Repository in Kubeapps to include the credentials: `http://<user>:<password>@<release_name>.<namespace>:8080`
+- Using a [JWT token](https://github.com/chartmuseum/auth-server-example). Once you obtain a valid token you can set it in the Authorization Header field of the App Repository form.
+
+## Artifactory
+
+JFrog Artifactory is a Repository Manager supporting all major packaging formats, build tools and CI servers.
+
+> **Note**: In order to use the Helm repository feature, it's necessary to use an Artifactory Pro account.
+
+To use Artifactory with Kubeapps the first thing is adding the JFrog repository to Kubeapps. Go to `Configuration > App Repositories` and add their repository:
+
+<img src="../img/jfrog-repository.png" alt="JFrog repository" width="400px">
+
+Then click on the JFrog repository and deploy Artifactory. For detailed installation instructions, check its [README](https://github.com/jfrog/charts/tree/master/stable/artifactory). If you don't have any further requirement, the values by default will work.
+
+When deployed, during the first login, select "Helm" to initialize a repository:
```

Comment:
what do you mean "during the first login", is that the setup wizard?

Reply:
yep